domains/zenfacture.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "iagoatbest26benbien",
+    "email": "ton@email"
```

you can safely delete your "invalid email" 
```suggestion
    "username": "iagoatbest26benbien"
```